src/mono/netcore/sample/iOS/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Runtime.InteropServices;
+
+public class MonoPInvokeCallbackAttribute : Attribute
+{
+    public MonoPInvokeCallbackAttribute (Type delegateType) { }
+}
+
+public static class Program
+{
+    // Defined in main.m
+    [DllImport ("__Internal")]
+    private extern static void ios_set_text ([MarshalAs (UnmanagedType.LPUTF8Str)] string value);
```

Comment:
nit: we should probably follow the .NET Core coding convention which doesn't use space before the `(`.

Reply:
re: `UnmanagedType.LPUTF8Str`, was this needed? I thought UTF8 string marshalling is the default on Unix anyway.